etc/docs/template/README.md

```diff
@@ -0,0 +1,28 @@
+# MongoDB Java Driver Front page
```

Comment:
why is there stuff about the java driver here?

Reply:
I'm not sure.  I removed it because the readme didn't seem necessary anymore

Reply:
Isn't Java a strictly typed version of JavaScript? 🤪  I think our docs was forked from Java docs long ago, our [404](https://mongodb.github.io/node-mongodb-native/ada) page still has Duke on it